server/utils/utils.test.ts

```diff
@@ -48,594 +49,638 @@ import {
 import { ReferenceCode } from '../interfaces/prisonApi/referenceCode'
 import { CommunityManager } from '../interfaces/prisonerProfileDeliusApi/communityManager'
 
-describe('convert to title case', () => {
-  it.each([
-    [null, null, ''],
-    ['empty string', '', ''],
-    ['Lower case', 'robert', 'Robert'],
-    ['Upper case', 'ROBERT', 'Robert'],
-    ['Mixed case', 'RoBErT', 'Robert'],
-    ['Multiple words', 'RobeRT SMiTH', 'Robert Smith'],
-    ['Leading spaces', '  RobeRT', '  Robert'],
-    ['Trailing spaces', 'RobeRT  ', 'Robert  '],
-    ['Hyphenated', 'Robert-John SmiTH-jONes-WILSON', 'Robert-John Smith-Jones-Wilson'],
-  ])('%s convertToTitleCase(%s, %s)', (_: string, a: string, expected: string) => {
-    expect(convertToTitleCase(a)).toEqual(expected)
+describe('utils', () => {
```

Comment:
Sorry just introduced an additional 'describe' block to make it easier to run all tests within IDE, so GitHub thinks all of this file has changed... I'll put another comment where the actual change has been introduced...

Reply:
Oo useful thank you!